config/spryk/spryks/Spryker/Zed/Business/AddZedBusinessFactoryGetProvidedDependencyMethod.yml

```diff
@@ -0,0 +1,86 @@
+spryk: method
+description: "Adds a get provided dependency method to the *module*BusinessFactory."
+mode: both
+
+arguments:
+    module:
+        inherit: true
+
+    organization:
+        inherit: true
+        default: Spryker
+
+    constantName: ~
+
+    output: ~
+
+    body:
+        value: "return $this->getProvidedDependency(\\{{ organization }}\\Zed\\{{ module }}\\{{ module }}DependencyProvider::{{ constantName }});"
+
+    factoryMethod:
+        callbackOnly: true
+        callback: ZedBusinessFactoryMethodNameCallback
+
+    template:
+        value: Common/Method.php.twig
+
+    target:
+        value: "{{ organization }}\\Zed\\{{ module }}\\Business\\{{ module }}BusinessFactory"
+
+preSpryks:
```

Comment:
You need to add these to the `AddCrudFacade` spryk as it will use it as often as it likes.

When you add specific definitions with arguments this Spryk can't be re-used for other cases.